flink-ml-lib/src/main/java/org/apache/flink/ml/classification/knn/FastDistanceVectorData.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.ml.classification.knn;
+
+import org.apache.flink.ml.linalg.DenseVector;
+import org.apache.flink.util.Preconditions;
+
+import java.io.Serializable;
+
+/** Save the data for calculating distance fast. The FastDistanceMatrixData */
+public class FastDistanceVectorData implements Serializable, Cloneable {
+    /** Stores the vector(sparse or dense). */
```

Comment:
This class supports only `DenseVector` for now. Could we make the comment consistent with the implementation?

Reply:
This class has been deleted.